app/rest/rest/pom.xml

```diff
@@ -161,11 +161,36 @@
       <artifactId>jaxb-core</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>javax.ws.rs</groupId>
+      <artifactId>javax.ws.rs-api</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.jboss.resteasy</groupId>
+      <artifactId>resteasy-client</artifactId>
+    </dependency>
```

Comment:
It would be awesome if we would collect the provider for all metrics related UI queries into an common module `rest/metrics`, similarly to the rest/db-logging. 

Suggestion:

```
rest/metrics
     +--- jsondb (includes Collector and Metrics Provider for looking up in the DB)
     +--- prometheus (include the access to Prometheus)
     +--- api (the interface accessed to be the endpoint)
```

That way we could keep the resteasy client dependency out of the rest module (the same way as we did for accessing the 'verifier') as well as other dependencies.

The metrics rest endpoint then just dispatches to one of the implementations via a defined interface. This implementation is injected by the proper SpringBoot configs triggers.

// @KurtStam